apps/api/src/server/applications/s51advice/s51-advice.controller.js

```diff
@@ -438,3 +438,19 @@ export const removePublishItemFromQueue = async ({ body }, response) => {
 
 	response.send(updatedS51Advice);
 }
+
+/**
+ * Checks whether passed s51 title is unique to this case. Test is case-insensitive, and search string is trimmed.
+ *
+ * @type {import('express').RequestHandler<{id: number, title: string}, any, any, any>}
+ */
+export const verifyS51TitleIsUnique = async ({ params }, response) => {
+	const { id, title } = params;
```

Comment:
Does id need to be converted to a Number here?

Reply:
good call - already caught in the validation middleware beforehand